src/wallet/coinselection.h

```diff
@@ -197,6 +197,32 @@ struct OutputGroup
  */
 [[nodiscard]] CAmount GetSelectionWaste(const std::set<CInputCoin>& inputs, CAmount change_cost, CAmount target, bool use_effective_value = true);
 
+struct SelectionResult
+{
+    /** Set of inputs selected by the algorithm to use in the transaction */
+    std::set<CInputCoin> m_selected_inputs;
```

Comment:
Should this be `private`? Especially since you have functions to `AddInput()` and `GetInputVector()`, nobody should have direct access to this set.

Reply:
Inserted a commit which makes them private.